packages/config/src/mutations/update.js

```diff
@@ -10,6 +10,26 @@ import { serializeToml } from '../utils/toml.js'
 
 import { applyMutations } from './apply.js'
 
+// Performs a merge of the configuration changes with the existing `netlify.toml`,
+// and returns the updated configuration
+export const mergeConfig = async function (
```

Comment:
```suggestion
/**
 * Performs a merge of the configuration changes with the existing `netlify.toml`,
 * and returns the updated configuration
 */
export const mergeConfig = async function (
```

Use JSDoc for function annotations as they can be picked up by the intellisense